src/App.js

```diff
@@ -59,6 +60,7 @@ class App extends Component {
           </div>
           <Route path='/collab/:room?' render={(props) => (<Collab {...props} userName={localStorage.userName} />)} />
           <Route path='/register' component={Register} />
+          <LandingPage />
```

Comment:
This needs to be <Route path='/' component={LandingPage} />, and a corresponding link needs to be made up above for the two links back to the root page, so that the landing page can be hit again after hitting the specified route

Reply:
Needs to be a route similar to the others (the code was taken out since it was wrapped in <> in the comment above, sorry)